palisade-service/README.md

```diff
@@ -34,8 +41,7 @@ to the Filtered Resource Service and retrieve the processed request.
 
 (fields marked with * are acquired from headers metadata)
 
-The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further
-processing.
+The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further processing.
```

Comment:
*token should be the first entry in the column PalisadeSystemResponse
change PalisadeResponse to PalisadeClientResponse
No sure what to suggest in wording, but the current wording mentions taking information from the PalisadeRequest and the table shows the PalisadeSystemResponse object.  Suggest re-gigging this so the reader will know that the PalisadeSystemResponse represents the Kafka message that will be sent to the User Service

Reply:
updated